### DISCUSSION TITLE
Competition and inspiration

### DISCUSSION THREAD

**Discussion by SelinaBuff** [inovex/scrumlr.io#4194] [General]
Here we can document all the retro tools we come across. Perhaps some of these tools will provide us with some inspiration.

**Comment by SelinaBuff** [comment]
https://www.parabol.co/agile/retrospectives/

**Reply by Schwehn42** [reply]
i like that this features integrations with Jira/Github, something that is already discussed in #4119.

**Comment by SelinaBuff** [comment]
https://www.stackfield.com/de/

**Comment by mateo-ivc** [comment]
Found a list with free and non-free retro-services:
[https://github.com/josephearl/awesome-retrospectives](https://github.com/josephearl/awesome-retrospectives)

**Reply by Schwehn42** [reply]
nice, seems like @brandstetterm got us added to that list :D https://github.com/josephearl/awesome-retrospectives/pull/16

**Reply by brandstetterm** [reply]
Yes, and according to our analytics, 270 people have already clicked on it since then 🥳
